frontend-ui/src/views/PipelineView.vue

```diff
@@ -167,8 +170,29 @@ async function loadData(limit: number, skip: number, filter?: string) {
 }
 
 async function loadLastRuns() {
-  // TODO: Load last runs
+  const schedule_names = Array.from(new Set(tasks.value.map((task) => task.schedule_name)))
+  loadingStore.startLoading('Loading last runs...')
```

Comment:
I just realize that this means we cannot "use" the UI until all last runs are loading ... this is not optimal (especially if there are 200 of them). We can postpone this to a later enhancement (meaning we can merge the PR without it), but I would appreciate if we could make the UI immediately available, in many cases we do not mind about last runs.